ale/drivers/lro_drivers.py

```diff
@@ -220,3 +221,202 @@ def sampling_factor(self):
           Number of samples and lines combined from the original data to produce a single pixel in this image
         """
         return self.crosstrack_summing
+
+class LroLrocIsisLabelNaifSpiceDriver(LineScanner, NaifSpice, IsisLabel, Driver):
+    @property
+    def instrument_id(self):
+        """
+        The short text name for the instrument
+
+        Returns an instrument id uniquely identifying the instrument. Used to acquire
+        instrument codes from Spice Lib bods2c routine.
+
+        Returns
+        -------
+        str
+          The short text name for the instrument
+        """
+        id_lookup = {
+            "NACL": "LRO_LROCNACL",
+            "NACR": "LRO_LROCNACR"
+        }
+
+        return id_lookup[super().instrument_id]
+
+    @property
+    def sensor_model_version(self):
+        """
+        Returns ISIS instrument sensor model version number
+
+        Returns
+        -------
+        : int
+          ISIS sensor model version
+        """
+        return 2
+
+    @property
+    def usgscsm_distortion_model(self):
+        """
+        The distortion model name with its coefficients
+
+        LRO LROC NAC does not use the default distortion model so we need to overwrite the
+        method packing the distortion model into the ISD.
+
+        Returns
+        -------
+        : dict
+          Returns a dict with the model name : dict of the coefficients
+        """
+
+        return {"lrolrocnac":
+                {"coefficients": self.odtk}}
+
+    @property
+    def odtk(self):
+        """
+        The coefficients for the distortion model
+
+        Returns
+        -------
+        : list
+          Radial distortion coefficients. There is only one coefficient for LROC NAC l/r
+        """
+        return spice.gdpool('INS{}_OD_K'.format(self.ikid), 0, 1).tolist()
+
+    @property
+    def light_time_correction(self):
+        """
+        Returns the type of light time correciton and abberation correction to
+        use in NAIF calls.
+
+        LROC is specifically set to not use light time correction because it is
+        so close to the surface of the moon that light time correction to the
+        center of the body is incorrect.
+
+        Returns
+        -------
+        : str
+          The light time and abberation correction string for use in NAIF calls.
+          See https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/req/abcorr.html
+          for the different options available.
+        """
+        return 'NONE'
+
+    @property
+    def detector_center_sample(self):
+        """
+        The center of the CCD in detector pixels
+        ISIS uses 0.5 based CCD samples, so we need to convert to 0 based.
+
+        Returns
+        -------
+        float :
+            The center sample of the CCD
+        """
+        return super().detector_center_sample - 0.5
```

Comment:
Does this make sense to have as a default at the `IsisLabel` level?

Reply:
This stuff is inconsistent between instruments. Some missions use ISIS pixels, some use 0 based pixels.

Reply:
By "ISIS Pixels" here, we just mean 0.5-based? I was looking at the comment and not what the code actually does and taking it too literally: "ISIS uses 0.5-based so we need to convert to 0-based" to me implies that we're somehow converting from "ISIS pixels" to "0-based pixels" but of course there are no real, literal "ISIS Pixels" here, it's just that the IK for the instrument happens to use 0.5-based pixels and we need 0-based pixels. Right?

Reply:
@kberryUSGS yes that is correct. This is converting whatever the IK uses, not what ISIS uses. Maybe a better comment is in order.

Reply:
@jessemapel I don't think that an updated comment needs to be a change on this PR -- this comment is used in many places -- this could be a ticket to either update wording or define "ISIS pixels" to mean 0.5-based pixels in general somewhere.